xs/src/libslic3r/PrintConfig.cpp

```diff
@@ -1392,6 +1392,16 @@ PrintConfigDef::PrintConfigDef()
     def->enum_labels.push_back("0.2 (detachable)");
     def->default_value = new ConfigOptionFloat(0.2);
 
+    def = this->add("support_material_max_layers", coInt);
+    def->label = "Max layer count for supports";
+    def->category = "Support material";
+    def->tooltip = "Do not print supports above this layer (0 is disabled). This differs from the Enforce-layers option in a couple main ways: 1. This option will do nothing if Supports are disabled, it merely terminates generation above the selected height. 2. The Enforce option does not honor some other support option restrictions, in particular the Overhang threshold, while this Max layer count still honors those. Default is 0 (limit disabled).";
```

Comment:
This seems overly wordy and includes unrelated information.

Suggest revising to:
"Disable support generation above this layer. A value of 0 means that all layers are considered for supports."